app/src/main/java/com/babylon/wallet/android/data/gateway/extensions/StateNonFungibleDetailsResponseItemExtensions.kt

```diff
@@ -56,7 +57,7 @@ private fun ProgrammaticScryptoSborValue.toMetadata(isCollection: Boolean = fals
                 // when the key is in the list of explicitly handled NFT data
                 MetadataType.String
             } else {
-                if (sborValue.value.isValidUrl()) {
+                if (sborValue.value.isValidUrl() || sborValue.value.isValidUrlWithSpacesInLastPathSegment()) {
```

Comment:
I would add this case maybe only for `key == key_image_url` not sure if we want to add this check for all sbor values. 

What if the `key = description` and the `value = https://somewhere.com/page description` This should not become a url.

So I think we should make the check to be strict for general keys but make the check less strict when `key = key_image_url`

Reply:
makes sense, will update

Reply:
Let's wait if this is indeed a bug.